pkgs/applications/misc/kjv/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, gawk, gnutar, gnumake, coreutils, gzip }:
+
+stdenv.mkDerivation rec {
+  name = "kjv";
+  version = "1.0.0";
```

Usually you want to keep with versions that they release, if they haven't (like in this case), then just give it the date of the last update
```suggestion
  version = "20190311";
```